tools/run-dev.py

```diff
@@ -147,7 +147,8 @@
 if not os.path.exists("node_modules/.bin/handlebars") and os.path.exists("node_modules/handlebars"):
     print("Handlebars binary missing due to rebase past .gitignore fixup; fixing...")
     subprocess.check_call(["rm", "-rf", "node_modules/handlebars"])
-    subprocess.check_call(["yarn", "install", "--non-interactive", "--prefer-offline"])
+    YARN_BIN = "/srv/zulip-npm-cache/yarn/bin/yarn"
```

Comment:
There we should probably just call yarn. In general I think there is a trade off between simplicity of the code and possibly using the wrong version of yarn if the use has a different version installed.